src/restartscript.cpp

```diff
@@ -528,7 +530,8 @@ void writeScript(const string& ckptDir,
                   "        . $DMTCP_SRUN_HELPER_SYNCFILE\n"
                   "        pass_slurm_helper_contact \"$DMTCP_LAUNCH_CKPTS\"\n"
                   "        rm $DMTCP_SRUN_HELPER_SYNCFILE\n"
-                  "        dmtcp_restart --join --coord-host $DMTCP_COORD_HOST"
+                  "        dmtcp_restart --join-coordinator"
+                              " --coord-host $DMTCP_COORD_HOST"
```

Comment:
Let's leave it as if for 2.5. For master, we can do the following:
```
"        dmtcp_restart --join-coordinator \\\n"
"          --coord-host $DMTCP_COORD_HOST \\\n"
"          --coord-port $DMTCP_COORD_PORT \\\n"
"          $DMTCP_LAUNCH_CKPTS\n"
```

This is more readable in the source as well as the script.